uiapi/handlers.go

```diff
@@ -773,20 +779,28 @@ func (h handler) GetAnnualSummary(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro orgao=%s inválido", agencyName))
 	}
 	host := c.Request().Host
-	strAgency.URL = fmt.Sprintf("%s/v1/orgao/%s", host, strAgency.ID)
+	strAgency.URL = fmt.Sprintf("%s/v2/orgao/%s", host, strAgency.ID)
 	summaries, err := h.client.GetAnnualSummary(agencyName)
 	if err != nil {
 		log.Printf("error getting annual data of '%s' :%q", agencyName, err)
 		return c.JSON(http.StatusInternalServerError, fmt.Sprintf("Algo deu errado ao tentar coletar os dados anuais do orgao=%s", agencyName))
 	}
 	var annualData []annualSummaryData
 	for _, s := range summaries {
+		baseRemPerMonth := s.BaseRemuneration / float64(s.NumMonthsWithData)
+		baseRemPerCapita := s.BaseRemuneration / float64(s.NumMonthsWithData) / float64(s.Count)
```

Comment:
Como conversamos via chat, não é assim que realizaremos estes cálculos. Se for anual, precisamos somar o total de membros de todos os meses.